compiler/python_archive.py

```diff
@@ -172,6 +179,7 @@ def generate_boilerplate(self, import_roots):
             'runtime_package': _runtime_package,
             'import_roots': str(import_roots),
             'zip_safe': self.zip_safe,
+            'extract_dir': self.extract_dir.replace('"', '\\"'),
```

Comment:
Is there a stdlib we could use to escape this rather than just replacing double quotes? i.e. `re.escape` or similar